index.html

```diff
@@ -5,22 +5,143 @@
     	<meta name="description" content="Your description goes here">
     	<meta name="keywords" content="one, two, three">
 
-		<title>Blank Template</title>
+		<title>Relaxr</title>
 
 		<!-- external CSS link -->
-		<link rel="stylesheet" href="css/reset.css">
+		<link rel="stylesheet" href="css/normalize.css">
+		<link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,600,800" rel="stylesheet">
 		<link rel="stylesheet" href="css/style.css">
 
 	</head>
 	<body>
-		<div id="container">
-			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
+		<div>
+			<header class="clearfix">
+				<h1 class="logo">Relaxr</h1>
+			
+				<nav>
+					<a href="#">About</a>
+					<a href="#">FAQ</A>
+					<a href="#">Team</a>
+					<a href="#">Contact Us</a>
+					<a href="#">Blog</a>
+				</nav>
 			</header>
+		</div>
+
+			<aside class="sidebar">
```

Comment:
There are few issues with your code including the use of "float: right;" which we don't recommend using except in a few small instances. You should be using "float: left;" when creating columns.

I would recommend looking back at Eric's guide for creating columns. You're missing a few critical elements such as your clearfix and your floated elements do not have a parent container.

We should take some time to go over this homework assignment together. Let me know when you can schedule some office hours.